usaspending_api/config/envs/default.py

```diff
@@ -212,18 +214,44 @@ def _validate_database_url(cls, values, db_url_conf_name, db_conf_prefix, requir
                     err_msg += f"\tPart: {k}, Part Value Provided: {v[0]}, Value found in {db_url_conf_name}: {v[1]}\n"
                 raise ValueError(err_msg)
 
+    # noinspection PyMethodParameters
+    # Pydantic returns a classmethod for its validators, so the cls param is correct
     @root_validator
-    def _DATABASE_URLs_and_parts_factory(cls, values):
+    def _DATABASE_URL_and_parts_factory(cls, values):
         """A root validator to backfill DATABASE_URL and USASPENDING_DB_* part config vars and validate that they are
-        all consistent. Similarly handles DATA_BROKER_DATABASE_URL and BROKER_DB_* part config vars.
+        all consistent.
 
         - Serves as a factory function to fill out all places where we track the database URL as both one complete
         connection string and as individual parts.
         - ALSO validates that the parts and whole string are consistent. A ``ValueError`` is thrown if found to
         be inconsistent, which will in turn raise a ``pydantic.ValidationError`` at configuration time.
         """
-        cls._validate_database_url(cls, values, "DATABASE_URL", "USASPENDING_DB", required=True)
-        cls._validate_database_url(cls, values, "DATA_BROKER_DATABASE_URL", "BROKER_DB", required=False)
+        # noinspection PyArgumentList
+        cls._validate_database_url(
+            cls=cls, values=values, db_url_conf_name="DATABASE_URL", db_conf_prefix="USASPENDING_DB", required=True
+        )
+        return values
+
+    # noinspection PyMethodParameters
+    # Pydantic returns a classmethod for its validators, so the cls param is correct
+    @root_validator
+    def _DATA_BROKER_DATABASE_URL_and_parts_factory(cls, values):
```

Comment:
I'm fine with keeping this split. I think I merged them at one point worried that values was being populated going into the Broker call but that was an issue with testing on my part. I think this is fine.

Reply:
Yeah, this is me uncertain of how the pydantic framework discovers and processes validators (and the `values` return value from them). I thought if 2 values are set in `values`, to do it in two steps so each value being set is deliberate.